packages/rocketchat-statistics/server/lib/SAUMonitor.js

```diff
@@ -0,0 +1,410 @@
+import { Meteor } from 'meteor/meteor';
+import { Accounts } from 'meteor/accounts-base';
+import UAParser from 'ua-parser-js';
+import { UAParserMobile } from './UAParserMobile';
+import { Sessions } from 'meteor/rocketchat:models';
+import { Logger } from 'meteor/rocketchat:logger';
+import { SyncedCron } from 'meteor/littledata:synced-cron';
+
+const getDateObj = (dateTime) => {
```

Comment:
```suggestion
const getDateObj = (dateTime = new Date()) => {
```

then the `if` below can be removed